src/gen_dragonfruit/gd_dev_adafruit_MPL115A2_temp.cpp

```diff
@@ -10,7 +10,7 @@ void gd_dev_adafruit_MPL115A2_temp_open(void){
 int32_t gd_dev_adafruit_MPL115A2_temp_read(void){
```

Comment:
Do you need a signed integer since you changed to K? Do you need the full 32 bits to store the range that you're expecting?